opencog/cython/opencog/Cast.cc

```diff
@@ -8,3 +8,4 @@ using namespace opencog;
 Handle atom_from_the_void(long p) { return *((Handle*) p); }
 long void_from_candle(const Handle& hp) { return (long) (&hp); }
 long void_from_cptr(Handle* hp) { return (long) (hp); }
+long avoid_from_cptr(Handle* hp) { return (long) (hp->operator->()); }
```

Comment:
Re UUID's, they are now handled as a utility in the "libatomspaceutils" directory.  Both the database bindings, and the REST server uses UUID's.